spec/features/ingredients/ingredients_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+RSpec.describe "the ingredients index" do
+# User Story 1 - Ingredients Index
+  it "displays ingredients" do 
+    ingredient1 = Ingredient.create!(name: "Pasta", cost: 2)
+    ingredient2 = Ingredient.create!(name: "Garlic", cost: 3)
+
+    visit "/ingredients"
+
+    expect(page).to have_content(ingredient1.name)
+    expect(page).to have_content(ingredient1.cost)
```

Comment:
You may want to consider using string labels for these attributes, so we (and the user) know waht they correspond to. Ex: 
```
have_content("Name: #{ingredient.name")
have_content("Cost: #{ingredient.cost")
```
And to punch it up even further, using a within block is a great way to keep index pages organized - and a great way to tell that this name and cost both belong to `"#ingredient-#{ingredient1.id}"`.